llmfoundry/utils/config_utils.py

```diff
@@ -184,3 +185,85 @@ def log_config(cfg: DictConfig) -> None:
             raise e
         if mlflow.active_run():
             mlflow.log_params(params=om.to_container(cfg, resolve=True))
+            log_dataset_uri(cfg)
+
+
+def parse_source_dataset(cfg: DictConfig):
+    """Parse a run config for dataset information."""
+    data_paths = set()
+
+    for data_split in ['train', 'eval']:
+        split = cfg.get(f'{data_split}_loader', {}).get('dataset',
+                                                        {}).get('split', None)
+        source_dataset_path = cfg.get(f'source_dataset_{data_split}', {})
+
+        # check for Delta table
+        if source_dataset_path and len(source_dataset_path.split('.')) >= 3:
+            data_paths.add(('delta_table', source_dataset_path, data_split))
+        # check for UC volume
+        elif source_dataset_path and source_dataset_path.startswith('/Volumes'):
```

Comment:
shouldn't the UC check be for dbfs:/? clearly what you did worked, so what am i missing?

Reply:
https://github.com/mlflow/mlflow/blob/e39dc8ca3438624a6ff1ea46bd9adf83e8355002/mlflow/data/uc_volume_dataset_source.py#L10

They just set it up that way in mlflow

Reply:
but isn't this checking against what we pass in? And we pass in a path starting with dbfs:/